templates/vertical-full-page-map/page.html.hbs

```diff
@@ -1,6 +1,8 @@
 {{#> layouts/html pageWrapperCss="YxtPage-wrapper--mobileListView" }}
   {{#> script/core }}
     {{> cards/all }}
+    {{> theme-components/theme-map/script}}
```

Comment:
did we just forget these before?

Reply:
oh I took a quick look and I think they already exist in page-setup?

Reply:
This PR moves them from page setup to here

Reply:
can we move them back into page-setup? Otherwise preexisting vertical-full-page-map pages that don't have these partials will break on upgrade.

Reply:
sure

Reply:
updated!